modules/ROOT/pages/tutorials/aura-query-api-tutorial.adoc

```diff
@@ -0,0 +1,79 @@
+[[aura-query-api-tutorial]]
+= Aura Query API
+:description: This page describes how to create a node using Query API and an Aura instance.
+
+== Introduction
+
+The Query API enables interaction with an Aura database using Cypher via HTTPS.
+
+This tutorial guides you through the process of sending Cypher queries to an Aura instance using the Query API.
+
+For more information, see the link:https://neo4j.com/docs/query-api/current/[Query API documentation] but note that:
+
+* For Aura instances, the host follows the format `<databaseID>.databases.neo4j.io`
+
+* The official documentation mainly refers to self-managed instances that use port `:7474` however Aura supports HTTPS, which defaults to port `:443` 
+
+* As HTTPS implies port `:443` in the request, you do not need to include the port in the request.
+
+== API endpoint
+
+To send Cypher queries to an Aura instance, make a `POST` request to the following API endpoint:
+
+[source, header]
+----
+https://<databaseID>.databases.neo4j.io/db/<databaseName>/query/v2
+----
+
+*Parameters*
+
+`<databaseID>` = The unique ID of your Aura database (e.g. e011c23c).
+
+`<databaseName>` = The target database (use `neo4j` for the default database).
+
+*Example Endpoint:*
+
+[source, header]
+----
+https://537dfda7.databases.neo4j.io/db/neo4j/query/v2
+----
+
+== Queries using curl
+
+You can interact with the Query API using curl, in your command-line terminal. 
+This provides a simple way to communicate with the database without using a driver. 
+Ensure that curl is installed in your terminal before proceeding.
+
+== Example: create a node 
+
+This example demonstrates how to create a Person node in an Aura instance, using curl:
+
+[source, shell]
+----
+curl -vvv -X POST \
+   -H "Authorization: Basic bmVvNGo1MlFidnE4TEFrRnZwXzdjT1NRM2F1NTJXVVRpTXotUERXQTU1NTdmRWFKdw==" \
+   -H "Content-Type: application/json" \
+   -d '{ "statement": "create (n:Person)  RETURN n" }' \ 
+   https://537dfda7.databases.neo4j.io/db/neo4j/query/v2
+----
+
+=== Explanation
```

Comment:
I recommend using callouts directly in the example (`# <.>`). In this way you don't need to repeat all the parameters, and the distance between the example and its clarifications is shorter.

For an example, see https://github.com/neo4j/docs-drivers/blob/dev/python-manual/modules/ROOT/pages/query-simple.adoc?plain=1#L15